lambdas/db-migration/src/migrations/20201014105058_create_granules_table.ts

```diff
@@ -6,6 +6,14 @@ export const up = async (knex: Knex): Promise<void> =>
       .bigIncrements('cumulusId')
       .comment('Internal Cumulus ID for a granule')
       .primary();
+    table
+      .text('granuleId')
+      .comment('Granule ID')
+      .notNullable();
+    table
+      .enum('status', ['running', 'completed', 'failed'])
+      .comment('Ingest status of the granule')
+      .notNullable();
```

Comment:
Is there not a reason to use snake case here and convert the rest of the table names to use snake case as well?

Reply:
No, I plan on doing that once @nemreid's PR is merged, since he already renamed all of these fields